docs/docs/command-line-interface.mdx

```diff
@@ -80,22 +80,45 @@ these, `rasa train` will fall back to one of these commands by default.
 
-Use `rasa train finetune` if the model should not be trained from scratch but if you
-want to train a new model by improving the latest model in the `models/` directory.
-If have a specific model which you want to improve you may specify the path to this by
-running `rasa train finetune <path to model to finetune>`. Finetuning a model usually
-requires less epochs. It's suggested to either use a model configuration for finetuning
-which defines less epochs for the pipelines and policies or use the flag
-`--epoch-fraction`. `--epoch-fraction` will use a fraction of the epochs which are
-specified in the given model configuration file. If the `DIETClassifier` is for example
-configured to use 100 epochs, `--epoch-fraction 0.5` will only use 50 epochs for
-finetuning.
-
 The following arguments can be used to configure the training process:
 
 ```text [rasa train --help]
 ```
 
+### Incremental training
+
+In order to improve the performance of an assistant, it's helpful to practice [CDD](./conversation-driven-development.mdx)
+and add new training examples based on how your users have talked to your assistant. You can use `rasa train --finetune`
+to initialize the pipeline with an already trained model and further finetune it on the
+new training dataset consisting of the additional training examples. This will help reduce the
+training time of the new model.
+
+By default, the command picks up the latest model in the `models/` directory. If you have a specific model
+which you want to improve, you may specify the path to this by
+running `rasa train --finetune <path to model to finetune>`. Finetuning a model usually
+requires less number of epochs to train ML components like `DIETClassifier`, `ResponseSelector` and `TEDPolicy`.
+It's suggested to either use a model configuration for finetuning
+which defines lesser number of epochs than before or use the flag
+`--epoch-fraction`. `--epoch-fraction` will use a fraction of the epochs specified for each ML component
+in the given model configuration file. For e.g., if the `DIETClassifier` is configured to use 100 epochs,
+specifying `--epoch-fraction 0.5` will only use 50 epochs for finetuning.
+
+You can also finetune an only NLU and only Core model by using
+`rasa train nlu --finetune` and `rasa train core --finetune` respectively.
+
+Conditions for a model to be finetunable are as follows:
+
+1. The current configuration supplied should be exactly the same as the
+configuration used to train the model which is being finetuned.
+The only parameter that you can change is `epochs` for the individual ML components of the pipeline.
+
+2. The domain of the assistant hasn't changed which means no new
+intents, actions, entities and slots have been added to the training data.
+Adding new training examples for each of these is valid.
+
+3. The model to be finetuned is trained with `MINIMUM_COMPATIBLE_VERSION` of the currently installed rasa version.
```

Comment:
Do users have visibility on `MINIMUM_COMPATIBLE_VERSION`? If not, how do we specify this better?

Reply:
good question - I suspect only by looking at the source code...
Could change it to:
```
The model to be finetuned is trained with a version of Rasa compatible with the currently installed version.
```
Or do you think we need to be more specific?

Reply:
I mean the question really is how do they know if the current installed version is compatible  with the old version used for training.
Question: Do we increase the `MINIMUM_COMPATIBLE_VERSION` for every minor version?

Reply:
> Question: Do we essentially increase the MINIMUM_COMPATIBLE_VERSION for every new minor version? 

No, but only when there are breaking model changes (e.g. a new default action etc.)

Reply:
And good point re the MIN version. They only see this through the changelog or when running into an error when running the model.

Reply:
Does it then make sense to include this information in `rasa --version` command? Maybe as a separate PR after the release if it takes too much time?

Reply:
yes, great idea!

Reply:
Created an issue here - https://github.com/RasaHQ/rasa/issues/7477
For now, do you suggest re-wording the above line differently?

Reply:
I'd either leave it or say `latest minor` - it's technically incorrect but easy to understand and to check

Reply:
Okay, i'll leave it as is for now. Will add a comment in the final PR indicating that we an issue to make this easily visible.